coverage/block-navigation.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```

Comment:
I know this is already merged, but you should probably add this folder to the git ignore file, AFAIK it's not adding any meaningful info to the repo, it's only useful for whoever is running it.